pkg/util/util.go

```diff
@@ -25,31 +25,9 @@ import (
 	"slices"
 	"strings"
 
-	"k8s.io/apimachinery/pkg/util/rand"
-	"k8s.io/apimachinery/pkg/util/sets"
-
 	cmapi "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
 )
 
-// Deprecated: this function is no longer supported and will be removed in the future
-func OnlyOneNotNil(items ...interface{}) (any bool, one bool) {
-	oneNotNil := false
-	for _, i := range items {
-		if i != nil {
-			if oneNotNil {
-				return true, false
-			}
-			oneNotNil = true
-		}
-	}
-	return oneNotNil, oneNotNil
-}
-
-// Deprecated: use slices#Equal instead
-func EqualSorted(s1, s2 []string) bool {
-	return slices.Equal(s1, s2)
-}
```

Comment:
You deprecated these in https://github.com/cert-manager/cert-manager/pull/6593 which was released in `v1.14.0` this month:
 * https://github.com/cert-manager/cert-manager/pull/6593

Reply:
I would be very surprised if someone imported these functions.
So I think it is fine to remove them this soon.